integration_tests/src/test/resources/int_struct_formatted_overflows.json

```diff
@@ -0,0 +1,2 @@
+{"data": {"B": 99999999999999999999, "A": -9999999999999999999}}
+{"data": {"A": 0, "B": "0"}}
```

Comment:
Why 0 and 0? Did you want something that would not overflow just as a reference?

Reply:
Ah, right.  I wonder if this test file should be called `int_struct_formatted_problematic_rows`, instead of `*overflow`.

The second row is one of the corner cases suggested in [#11711](https://github.com/NVIDIA/spark-rapids/issues/11711#issuecomment-2471249997).  This also produces a different output row, though likely not because of an overflow:

On 3.5.3:
```
+----------------------------------------------------------------+---------------+
|json                                                            |from_json(json)|
+----------------------------------------------------------------+---------------+
|{"data": {"A": 0, "B": "0"}}                                    |{{0, NULL}}    |
+----------------------------------------------------------------+---------------+
```

On Databricks 14.3:
```
+----------------------------------------------------------------+---------------+
|json                                                            |from_json(json)|
+----------------------------------------------------------------+---------------+
|{"data": {"A": 0, "B": "0"}}                                    |{NULL}         |
+----------------------------------------------------------------+---------------+
```

Reply:
(To be clear, it's the quotes: `"0"` instead of `0`.  That borks up the parsing.)